// code derived form: https://github.com/erming/shout/
// reference:
//  https://github.com/erming/shout/commit/7ef2da0c832175cea1776a651a3c14051468fdc2

// https://github.com/erming/shout/blob/master/LICENSE
/*
The MIT License (MIT)

Copyright (c) 2014 Mattias Erming and contributors

Permission is hereby granted, free of charge, to any person obtaining a copy
of this software and associated documentation files (the "Software"), to deal
in the Software without restriction, including without limitation the rights
to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
copies of the Software, and to permit persons to whom the Software is
furnished to do so, subject to the following conditions:

The above copyright notice and this permission notice shall be included in
all copies or substantial portions of the Software.

THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
THE SOFTWARE.
*/

diff --git a/client/views/toggle.tpl b/client/views/toggle.tpl
index 08de2b5..629d821 100644
--- a/client/views/toggle.tpl
+++ b/client/views/toggle.tpl
@@ -9,7 +9,7 @@
 			{{#if thumb}}
 				<img src="{{thumb}}" class="thumb">
 			{{/if}}
-			<div class="head">{{{head}}}</div>
+			<div class="head">{{{parse head}}}</div>
 			<div class="body">
 				{{body}}
 			</div>
